src/snowcli/cli/common/flags.py

```diff
@@ -81,3 +83,26 @@
     callback=ConnectionDetails.update_callback("warehouse"),
     show_default=False,
 )
+
+OutputFormatOption = typer.Option(
+    OutputFormat.TABLE.value,
+    "--format",
+    help="Specifies output format",
+    case_sensitive=False,
+    callback=update_callback("output_format"),
+)
+
+VerboseOption = typer.Option(
+    None,
+    "--verbose",
+    "-v",
+    help="Print logs from level info and higher",
+    callback=update_callback("verbose"),
```

Comment:
should use here `is_flag=True` to indicate this option is flag?

Reply:
I will add